source/agora/common/Set.d

```diff
@@ -79,6 +79,12 @@ public struct Set (T)
             this.put(key);
     }
 
+    /// Returns true if the set has the element e, otherwise returns false
+    public auto opBinary (string op)(in T e) nothrow const pure @safe @nogc if (op == "in")
```

Comment:
not sure I am following this:
'Also it's a template so you don't need to specify all attributes.'

are those attributes stripped after the template instantiation? or are we expecting 
```D
return e in _set;
```
might throw for some instantiation?

Reply:
I mean that you can remove `@safe` because the compiler will check the body, see that it is `@safe`, and mark it `@safe` itself without you needing to do anything.